src/index.html

```diff
@@ -1,12 +1,374 @@
+<!-- #region header -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"  rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="catalog__header header">
+        <a href="#home">
+          <img
+            class="header__logo"
+            src="images/logo.png"
+            alt="moyo"
+           >
+        </a>
+        <nav>
+          <ul class="header__nav nav">
+            <li class="nav__link is-active" href="#">Apple</li>
```

Comment:
The li tag does not have the href attribute, since it is a link to the menu, so the text must be wrapped in the a tag.
Need to fix it everywhere
```suggestion
            <li class="nav__item">
              <a class="nav__link is-active" href="#">Apple</a>
            </li>
```